_docs/style/colors.md

```diff
@@ -11,20 +11,57 @@ status: complete
 {% include toc.html %}
 
 <section class="static-section"  markdown="1">
-### Primary Colors
 
-Primary colors should be used sparingly for primary and secondary actions,
-alerts, or data visualizations.
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## The Helix color palette
```

Comment:
Re: line 6 above...

Should we add something about color families? just to set the right expectation for the content that comes next. 

or maybe we need a brief intro  below the header but above the full palette?

Reply:
I think we can move the header content so that it is the intro. In one site, there is no header content.